PlaywrightSolar/Tests/ShoppingCardTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using PlaywrightSolar.PageObjects;
+
+namespace PlaywrightSolar.Tests;
+
+internal class ShoppingCardTest : UiTestFixture
+{
+    private InvertersPage _invertersPage;
+    private BasketPage _basketPage;
+    private SolarShopPage _solarShopPage;
+    
+    [SetUp]
+    public void SetupPages()
+    {
+        _invertersPage = new InvertersPage(Page);
+        _basketPage = new BasketPage(Page);
+        _solarShopPage = new SolarShopPage(Page);
+    }
+
+    [Test]
+    public async Task AddInverterProductToShoppingCard()
+    {
+        // Arrange
+        await _invertersPage.GoToInvertersPage();
+        
+        // Act
+        await _invertersPage.GetProductFromList();
```

Comment:
due to wrong name test still has unclear steps "get product, place order, delete product"